backend/Match/views.py

```diff
@@ -40,9 +40,9 @@ class LaunchTestMatchView(APIView):
 	@method_decorator(must_be_authenticated)
 	def get(self, request):
 		token = MatchToken.objects.create_single_match_token(request.user, request.user)
-		match, player_left_side, player_right_side = GameSetupManager.create_match_and_its_players(token)
+		match, player_left_side, player_right_side = MatchSetupManager.create_match_and_its_players(token)
```

Comment:
same here, we are not actually catching an error that might happen in the create_match... function are we

Reply:
if the problem is with postgres, we can return the 502 specifically but in any other case it should be some 400 but I dont see the forbidden being the correct one here.